### ISSUE TITLE
After reading the pre-trained model(AVA) and running the validate function, plcc and srcc are incorrectly high

### ISSUE THREAD

**Issue by NurseryRhymeA** [woshidandan/Image-Aesthetics-Assessment#10]
thanks for your contribution.
I'm meeting a problem in AVA dataset. When I read the pre-trained weights  and validate on the AVA dataset, the resulting metrics are incorrectly high.

![初始权重](https://github.com/woshidandan/Image-Aesthetics-Assessment/assets/80157816/c23fa10b-be73-47d5-9c67-d57f42dcc7f1)

The code to read the weights is shown below, I made some modifications since it doesn't run directly on my environment.

`def start_train(opt):
    train_loader, val_loader, test_loader = create_data_part(opt)
    args, config = parse_option()
    print(f"Creating model:{config.MODEL.TYPE}/{config.MODEL.NAME}")
    model = build_model(config)

    if os.path.exists(config.MODEL.RESUME):
        print(config.MODEL.RESUME)
        checkpoint = torch.load(config.MODEL.RESUME)
        pre_weights = torch.load(config.MODEL.RESUME)
        # pre_weights = checkpoint['model']
        pre_dict = {}
        for k, v in pre_weights.items():
            if "cls_head" not in k:
                pre_dict[k] = v
        model.load_state_dict(pre_dict, strict=False)
        #直接读取
        model.load_state_dict(torch.load(opt['path_to_model_weight'], map_location='cuda:0'))
    model.to('cuda')`

I want to know if you have also encountered this situation. I am a beginner and the modified code may have some errors. I hope to get some suggestions from you.

**Comment by ybbthou8** [comment]
The SRCC (0.87) and PLCC (0.87) I inferred on the AVA test set using the official AVA model are even one point higher than your results. Did you find the reason for this so that you closed it? I retrained using the AVA training set, but still couldn't achieve the results given by the official model on ava test set.

**Comment by NurseryRhymeA** [comment]
No. I followed the steps of the paper，used the pre-trained model on the ImageNet(dat_base_in1k_224.pth)  training set and retrained it on AVA Dataset. The final results are as follows:
acc: 0.81, plcc: 0.739, srcc: 0.73, loss: 0.072

**Comment by ybbthou8** [comment]
I directly retrain pretrained model on AVA use EMD loss and get plcc and srcc: 0.8/0.81在 2024年10月11日，14:47，NurseryRhyme ***@***.***> 写道：﻿
No. I followed the steps of the paper，used the pre-trained model on the ImageNet(dat_base_in1k_224.pth)  training set and retrained it on AVA Dataset. The final results are as follows:
acc: 0.81, plcc: 0.739, srcc: 0.73, loss: 0.072

**Comment by NurseryRhymeA** [comment]
> I directly retrain pretrained model on AVA use EMD loss and get plcc and srcc: 0.8/0.81在 2024年10月11日，14:47，NurseryRhyme ***@***.***> 写道：﻿ No. I followed the steps of the paper，used the pre-trained model on the ImageNet(dat_base_in1k_224.pth) training set and retrained it on AVA Dataset. The final results are as follows: acc: 0.81, plcc: 0.739, srcc: 0.73, loss: 0.072 —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

Is the pre trained model {dat_base_in1k_224.pth} or {AVA_AOT_vacc_0.8259_srcc_0.7596_vlcc_0.7710.pth}?If it is dat_base_in1k_224.pth, could you please provide the learning rate, batchsize and other hyperparameters during your training? Thank you very much！

**Comment by ybbthou8** [comment]
the pre-trained model weight is AVA_AOT_vacc_0.8259_srcc_0.7596_vlcc_0.7710.pth，AVA 1-10 score distribution，EMD loss，bs=64 and lr is 1e-4 without decay，20 epoch
发自我的 iPhone

> 在 2024年10月15日，20:16，NurseryRhyme ***@***.***> 写道：
> 
> AVA_AOT_vacc_0.8259_srcc_0.7596_vlcc_0.7710.pth}?If it is
